DemoViper/Sources/AppDelegate/RootRouter.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  RootRouter.swift
+//  DemoViper
+//
+//  Created by Issarapong Poesua on 8/10/19.
+//  Copyright © 2019 Nimble Co. Ltd. All rights reserved.
+//
+
+import UIKit
+
+final class RootRouter {
+    private let reposModule: ReposModule
+
+    init() {
+        reposModule = ReposModule()
+    }
+
+    func showRepos(on view: UIWindow) {
```

Comment:
You need to return module input here as well, so that app delegate can contact with module's presenter ✌️

```swift
    func showRepos(on view: UIWindow) -> ReposInput { 
        return reposModule.input
    }